ClosedXML_Tests/Excel/CalcEngine/StatisticalTests.cs

```diff
@@ -242,6 +242,40 @@ public void Init()
             workbook = SetupWorkbook();
         }
 
+        [Test]
+        public void Fisher_IncorrectCases()
+        {
+            var ws = workbook.Worksheets.First();
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(D3:D45)"),
+                Throws.TypeOf<CellValueException>().With.Message.EqualTo("Parameter non numeric."));
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(D3)"),
+                Throws.TypeOf<CellValueException>().With.Message.EqualTo("Parameter non numeric."));
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(H3)"),
+                Throws.TypeOf<NumberException>().With.Message.EqualTo("Incorrect value. Should be: -1 >= x <= 1."));
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(-1)"),
+                Throws.TypeOf<NumberException>().With.Message.EqualTo("Incorrect value. Should be: -1 >= x <= 1."));
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(1)"),
+                Throws.TypeOf<NumberException>().With.Message.EqualTo("Incorrect value. Should be: -1 >= x <= 1."));
```

Comment:
If `-1` and `1` are invalid arguments why there are `>=` and `<=` in the error message? I'd expect `>` and `<` there.

And yes, better make it a single test with multiple `TestCase`s.

Reply:
@Pankraty Yep. You are right about message. I fixed it and improve test for incorrect cases. Please check.